docs/api/saved-objects/resolve_import_errors.asciidoc

```diff
@@ -131,6 +174,21 @@ The API returns the following:
 --------------------------------------------------
 {
   "success": true,
-  "successCount": 1
+  "successCount": 2,
+  "successResults": [
+    {
+      "id": "my-pattern",
+      "type": "index-pattern"
+    },
+    {
+      "id": "my-dashboard",
+      "type": "dashboard"
+    }
+  ]
 }
 --------------------------------------------------
+
+This result indicates that the import was successful, and both objects were created.
+
+TIP: If a prior import attempt resulted in resolvable errors, you must include a retry for each object you want to import, including any
+that were described in the missing error object's `blocked` array. In this example, we retried importing the dashboard accordingly.
```

Comment:
One more comment!

This tip actually has a typo -- it should be `blocking` array, not `blocked` array. At any rate, I've removed the `blocking` array entirely in the client-side follow up to this PR, so I'll need to update the docs then to change the details on this.